modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Actions.tsx

```diff
@@ -152,51 +93,39 @@ const Actions = ({fdsView, namespace, spritemap}: IFDSViewSectionInterface) => {
 		setLoading(false);
 	};
 
-	const saveFDSAction = async () => {
-		setSaveButtonDisabled(true);
-
-		const body = {
-			[OBJECT_RELATIONSHIP.FDS_VIEW_FDS_ACTION_ID]: fdsView.id,
-			icon: iconSymbol,
-			type,
-			url,
-		} as any;
-
-		if (Liferay.FeatureFlags['LPS-172017']) {
-			body.confirmationMessage_i18n = confirmationMessageTranslations;
-			body.label_i18n = labelTranslations;
-		}
-		else {
-			body.confirmationMessage = confirmationMessage;
-			body.label = labelTranslations;
-		}
-
-		const response = await fetch(API_URL.FDS_ACTIONS, {
-			body: JSON.stringify(body),
-			headers: {
-				'Accept': 'application/json',
-				'Content-Type': 'application/json',
-			},
-			method: 'POST',
+	const deleteFDSAction = ({item}: {item: IFDSAction}) => {
+		openModal({
+			bodyHTML: Liferay.Language.get(
+				'are-you-sure-you-want-to-delete-this-action'
+			),
+			buttons: [
+				{
+					autoFocus: true,
+					displayType: 'secondary',
+					label: Liferay.Language.get('cancel'),
+					type: 'cancel',
+				},
+				{
+					displayType: 'danger',
+					label: Liferay.Language.get('delete'),
+					onClick: ({processClose}: {processClose: Function}) => {
+						processClose();
+
+						fetch(item.actions.delete.href, {
+							method: item.actions.delete.method,
+						})
+							.then(() => {
+								openDefaultSuccessToast();
+
+								getFDSActions();
```

Comment:
Similar as comment above. It is odd to "get" something but not do anything with it.